src/Playroom/CodeMirror-JSX.js

```diff
@@ -0,0 +1,306 @@
+/* eslint-disable new-cap */
```

Comment:
Is this file borrowed from somewhere else? Seems like a copy-and-paste.

Reply:
It was the addon-xml 😅 I went back for a second look and was able to use that code once it was registered to codemirror and just handle the tooltip stuff myself. 

I switched to rendering the tooltip with lit-html. I can remove that and switch back to document methods if you want, but I think the code is a little clearer this way.

Reply:
Actually I realized i could just use react and it become much clearer 👍